R/get.R

```diff
@@ -53,7 +53,7 @@ get <- function(value = NULL,
 
   # load the yaml
   config_yaml <- yaml::yaml.load_file(
-    file, eval.expr = TRUE
+    file, eval.expr = FALSE, handlers = list(expr = function(x) paste("!expr", x))
```

Comment:
I think this would be better as `function(x) parse(text = x)` — that way loading will still error if there are syntax errors in the R code.

Reply:
This is neat, thank you.